### PR TITLE
[Review Needed] Providing language selection at the beginning

### LINKED ISSUE #1034: Laguage Selection should be asked on loading of page rather than keeping on top bar
### How to use the Feature
Users will be able to select a language prior starting the game. This way if someone doesn't know English at all, the user will be able to change it in the beginning itself.

### How the Feature Improves the Game
This feature improves usability and understanding of the game for users who don't know English which is default.

### Implementation/Solution Ideas
We can use an overlay on loading of page containing options of selecting the language in the form a selection list.

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by Sangatdas**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at this [link](https://sangatdas.github.io/labyrinth/)

- Fixes #1034 

### The problem I want to solve or the facility I want to improve is/are
Language selection options should be provided before starting the game and not on the navbar.

### My steps to solve it

I have added an overlay which has the language options. I have made changes to custom.css, index.html and translation.js files.

### Screenshots,Path to Tile

screenshot from 2018-12-26 15-56-54

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
# :no_entry: More Work Needed
- [ ] TRAVIS FAILS. I think we should fix it. 


---
_Change the PR title from **[IN PROGRESS]** to **[REVIEW NEEDED]** when you have done the requested changes_
_please avoid pinging, we get notifications when an update is made_

**Comment by Sangatdas** [comment]
@YashKumarVerma Travis CI execute the command "bundle exec rake" even though its not included in travis.yml file.What possible solution can we have?

**Comment by Sangatdas** [comment]
@YashKumarVerma Travis has been failing for all PRs. Other than that check this PR out.

**Comment by YashKumarVerma** [comment]
Um, do we really need to change the build sript to add a section to the page ?

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,15 @@
+GEM
```
Comment (YashKumarVerma) [inline Gemfile.lock]: do we need this file ?

Comment (Sangatdas) [reply Gemfile.lock]: No, Gemfile.lock isn't required. Forgot to remove that.

Comment (Sangatdas) [reply Gemfile.lock]: @YashKumarVerma Have removed that in the pr now.

File: .travis.yml [hunk 2]
```diff
@@ -17,4 +17,4 @@ install:
 branches:
   only:
      - master
-     - /./
+- /./
```
Comment (YashKumarVerma) [inline .travis.yml]: do we need this ?

Comment (Sangatdas) [reply .travis.yml]: @YashKumarVerma Did requested changes.

**Comment by Sangatdas** [comment]
@abishekvashok Made requested changes. Review and merge.

**Comment by abishekvashok** [comment]
Thanks

**Comment by YashKumarVerma** [comment]
Good work

**Pull request merged**